src/output.jl

```diff
@@ -71,36 +72,41 @@ function savefields(file::JLD2.JLDFile{JLD2.MmapIO}, grid::TwoDGrid)
   for field in [:nx, :ny, :Lx, :Ly, :x, :y]
     savefield(file, "grid/$field", getfield(grid, field))
   end
-  nothing
+  
+  return nothing
 end
 
 function savefields(file::JLD2.JLDFile{JLD2.MmapIO}, grid::OneDGrid)
   for field in [:nx, :Lx, :x]
     savefield(file, "grid/$field", getfield(grid, field))
   end
-  nothing
+  
+  return nothing
 end
 
 function savefields(file::JLD2.JLDFile{JLD2.MmapIO}, params::AbstractParams)
   for name in fieldnames(typeof(params))
     field = getfield(params, name)
-    if !(typeof(field) <: Function)
+    if !(typeof(field) <: Function) && !(typeof(field) <: FFTW.FFTWPlan)
```

Comment:
You could make this design more sustainable by defining a variable

```
forbidden_output_types = [Function, FFTW.FFTWPlan]
```

and then changing the above line too

```
if !(any([typeof(field) <: forbidden for forbidden in forbidden_output_types]))
    # etc
```

Reply:
I might not have gotten the logic right there, but hopefully the idea is clear...

Reply:
Very clear and better indeed; will change it.